views/group/index.pug

```diff
@@ -1,8 +1,9 @@
 extends ../layouts/main
 
 block content
-  h1.title.is-1 Csoportok
-  a.button.is-link(href="/groups/new") Új csoport
+  .is-flex.flex-wrap.justify-between.align-center
+    h1.title.is-1.is-marginless Csoportok
+    a.button.is-link(href="/groups/new") Új csoport
```

Comment:
This flex thing is already implemented in bulma under the name "Columns": https://bulma.io/documentation/columns/basics/

I would have used them somewhat like this:
```
.columns.is-mobile.is-multiline.is-vcentered
    .column.is-narrow
      h1.title.is-1 Csoportok
    .column
      .buttons.is-right
        a.button.is-link(href="/groups/new") Új csoport
```
With this, the header does not look crammed on mobile (culprit: `.is-marginless` class), and there's even a gap between the button and title as a treat.
(Same for ticket/index)